src/components/AvatarWithDisplayName.js

```diff
@@ -90,21 +89,11 @@ function AvatarWithDisplayName(props) {
                             textStyles={[props.isAnonymous ? styles.headerAnonymousFooter : styles.headerText, styles.pre]}
                             shouldUseFullTitle={isExpenseReport || props.isAnonymous}
                         />
-                        {!_.isEmpty(parentNavigationSubtitle) && (
-                            <PressableWithoutFeedback
-                                onPress={() => {
-                                    Navigation.navigate(ROUTES.getReportRoute(props.report.parentReportID));
-                                }}
-                                accessibilityLabel={subtitle}
-                                accessibilityRole={CONST.ACCESSIBILITY_ROLE.LINK}
-                            >
-                                <Text
-                                    style={[styles.optionAlternateText, styles.textLabelSupporting, styles.link]}
-                                    numberOfLines={1}
-                                >
-                                    {parentNavigationSubtitle}
-                                </Text>
-                            </PressableWithoutFeedback>
+                        {hasParentNavigationSubtitle && (
+                            <ParentNavigationSubtitle
+                                parentNavigationSubtitleData={parentNavigationSubtitleData}
+                                report={props.report}
```

Comment:
The `accessibilityLabel` in `PressableWithoutFeedback` was `subtitle` but now it's `parentNavigationSubtitleData`. I don't think this a bug though. Just worth noting

Reply:
It's still the subtitle, I'm using the localize function

Reply:
It was this self subtitle:
`const subtitle = ReportUtils.getChatRoomSubtitle(props.report);`
Now it's the parent subtitle
`ReportUtils.getParentNavigationSubtitle(props.report);`

Reply:
I don't think this is necessary "wrong". Any will do, just wanted to highlight this change